.github/workflows/default.yml

```diff
@@ -14,11 +14,21 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v2
-      - name: Push to Docker Hub
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: gitbucket/gitbucket
-          tag_with_ref: true
-          dockerfile: Dockerfile
\ No newline at end of file
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x

```

Comment:
Do we actually need to build for all of these platforms?

Reply:
What if someone uses it? I use the arm64 version because of low resource consumption

Reply:
I would use amd64 and arm64, but not really sure about other platforms.